emergency_app/forms.py

```diff
@@ -0,0 +1,242 @@
+# Taken from this video:
+# https://www.youtube.com/watch?v=wVnQkKf-gHo
+from django import forms
+# from emergency_app.models.identity import Identity
+from emergency_app.models.contact import Contact
+from emergency_app.models.emergency import Emergency
+from emergency_app.models.nation import Nation
+# from emergency_app.models.state import State
+from common.util import sanitization
+
+# When required field=false, clean() would normalize empty value of CharField into empty string
+# However, null value was needed to follow the sample data provided. Therefore, this function is declared.
+# alternatively, to_python function for each field could use this function.
+# Source: https://docs.djangoproject.com/en/2.2/ref/forms/fields/
+def empty_string_handler(field):
+    # if field either empty string or empty/null value
+    if not field:
```

Comment:
Be careful here.  If non-string values could be provided, this will return None if the given value is 0 or False.  You could cast the value to a string for comparison just to be safe: `if field is None or not str(field):`